src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React from 'react'
+import { Routes, Route } from 'react-router-dom'
+import LoginSignup from './screens/LoginSignup'
+import Signup from './screens/Signup'
```

Comment:
I think the folder name "signup", instead if "Signup" is creating the build failure. Can you try renaming the folder once.